tests/mutmut_workflow.py

```diff
@@ -0,0 +1,65 @@
+import subprocess as sp
+import os
+
+import shelve
+import time
+
+files = ["executing/_position_node_finder.py","tests/deadcode.py"]
+
+py_version="py311"
+
+def mutmut_run(num: int | None = None):
+    cmd= [
+            "mutmut",
+            "run",
+            "--paths-to-mutate",
+            ",".join(files),
+            "--runner",
+            f"tox -e {py_version} -- --ff",
```

Comment:
```suggestion
            f"python{py_version} -m pytest --ff",
```

where `py_version = "3.11"`. `tox` can be a bit slow to start.

Reply:
tox provides a known environment with the correct dependencies. The startup time is not so important, because the mutmut run takes hours by itself.

mutmut and generate_small_samples are also using different python versions, because I had problems with mutmut on some python versions.

Reply:
> tox provides a known environment with the correct dependencies. The startup time is not so important, because the mutmut run takes hours by itself.

Surely mutmut is calling tox over and over again, so the startup slowdown is multiplied by the number of mutations it tests?

Reply:
there are ~500 mutations to test. My last mutmut run took ~4 days (generate_small_sample is limited to 1h).

I don't know how I could make this work without tox. I have to run a different python version with a specific environment (installed packages).